src/marqo/tensor_search/api.py

```diff
@@ -191,6 +193,29 @@ def memory():
     return memory_profiler.get_memory_profile()
 
 
+@app.post('/validate/{index_name}')
+@utils.enable_ops_api()
+def schema_validation(index_name: str, settings_object: str):
+    try:
+        validate_settings_object(index_name, settings_object)
+        return JSONResponse(
+            content={
+                "validated": True,
+                "index": index_name
+            },
+            status_code=200
+        )
+    except ValidationError as e:
+        return JSONResponse(
+            content={
+                "validated": False,
+                "validation_error": str(e),
```

Comment:
Should it be camelCase in the API?

Reply:
Yes API response and request JSON must be camel case